src/lib_ccx/ccx_encoders_smptett.c

```diff
@@ -263,11 +263,11 @@ int write_cc_buffer_as_smptett(struct eia608_screen *data, struct encoder_ctx *c
 					dbg_print(CCX_DMT_DECODER_608, "\r");
 					dbg_print(CCX_DMT_DECODER_608, "%s\n",context->subline);
 				}
+				#pragma GCC diagnostic ignored "-Wunused-variable" /* length is currently unused */
```

Comment:
If we're doing this here we could do it in the #ifdef DEBUG from above

Reply:
The difference is that these variables have no use even if something is defined.
By using #ifdef we show that these are necessary to debug.